index.js

```diff
@@ -1,7 +1,94 @@
 // Iteration 1: Names and Input
 
+const hacker1 = 'jason'
+console.log(`The driver's name is ${hacker1}`)
+const hacker2 = 'Michael'
+console.log(`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters`)
+} else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`)
+} else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
 
 // Iteration 3: Loops
+
+const letterUp = hacker1.toUpperCase();
+let up = '';
+for(let i=0; i<letterUp.length; i++){
+   up += letterUp[i] + ' ';     
+}
+console.log(up)
```

Comment:
Aqui el unico problema que tenemos (pijadas jejeje) es que si le digo mi nombre
CRIS
me va a devolver el ultimo elemento con el espacio tambien (voy a cambiar los espacios por guiones bajos para que se vean vale?)
C_R_I_S_
Ese ultimo elemento no deberia de estar, se te ocurre como hacerlo? :D

Reply:
let spaceName;
for(let i = 0; i < letterUp.length; i++){
   up += letterUp[i] + ' ' ;
}
spaceName = up.slice(0, up.length - 1);
console.log(spaceName);